### PR TITLE
Cray compiler fix

### PR DESCRIPTION
**Pull request by grantfirl**
- fix Cray compiler error having to do with binary/unary operator precedence standards
- add @AndersJensen-NOAA to CODEOWNERS for Thompson MP scheme

### CODE REVIEW

**Review by joeolson42 — APPROVED** [review]
Perfect. Thanks.

**Comment by grantfirl** [comment]
@joeolson42 Once you press the merge button here, https://github.com/ufs-community/ccpp-physics/pull/71 should automatically update. Then you'll need to update submodule pointers for your fv3atm and then ufs-weather-model PR branches.

**Pull request merged**
